sample_app/sampleapp_common.c

```diff
@@ -1214,19 +1217,21 @@ static void app_handle_cyclic_data (
       }
    }
 
-   /* Set LED state */
-   if (outputdata_is_updated == true && outputdata_iops == PNET_IOXS_GOOD)
+   if ((outputdata_length == APP_DATASIZE_OUTPUT) && (outputdata_iops == PNET_IOXS_GOOD))
    {
-      if (outputdata_length == APP_DATASIZE_OUTPUT)
+      /* Extract LED state from most significant bit */
+      received_led_state = (outputdata[0] & 0x80) > 0;
+
+      /* Set LED state */
+      if (received_led_state != previous_led_state)
       {
-         received_led_state = (outputdata[0] & 0x80) > 0; /* Use most
-                                                             significant bit */
          app_set_led (APP_DATA_LED_ID, received_led_state);
       }
-      else
-      {
-         printf ("Wrong outputdata length: %u\n", outputdata_length);
-      }
+      previous_led_state = received_led_state;
+   }
+   else
+   {
+      printf ("Wrong outputdata length: %u or IOPS: %u\n", outputdata_length, outputdata_iops);
```

Comment:
I can't see line length in github diff view. However this line is split by the diff viewer. I think this is an indication that it is too long according to coding standard..

Reply:
Done. Did run clang-format.